ko/admin/admin_utils.rst

```diff
@@ -1003,6 +1004,25 @@ plandump
 
         cubrid plandump -o output.txt testdb
 
+.. option:: -s, --sha1=SHA1
+
+    캐시에 저장된 특정 질의 수행 계획을 제거한다.::
```

Comment:
How about this?

> 입력한 SHA1 코드와 같은 코드를 가지고 있는 질의 수행 계획을 제거한다.::

Of course, by looking at the example, people can see that it is removed based on the SHA1 code, but it would be good to specify this.

Reply:
```suggestion
    캐시에 저장된 SHA1 코드의 특정 질의 수행 계획을 제거한다.::
```

Reply:
I fixed it.